torchtext/data/functional.py

```diff
@@ -282,3 +283,19 @@ def __getitem__(self, idx):
             return self._data[idx]
 
     return _MapStyleDataset(iter_data)
+
+
+def gpt2_bpe_tokenizer(input: str) -> List[str]:
```

Comment:
Yes, I was thinking that too. However, for the completeness of this PR, I wanted to make sure that we have tests and I observed that functionality exposed via `csrc` did not have its dedicated tests. What's your recommendation on that?

Reply:
It's a valid point, we do not have dedicated testing for operators and the testing is mainly done through python functions using them. let's have dedicated unit-testing for csrc operators. We can start by adding test for this operator. @mthrok I am curious, if there are dedicated unit-tests for operators in audio or is the testing is done indirectly via the APIs that are only exposed to the users?

Reply:
We do not have tests for C++ yet in audio, but we are in desperate need to setup. So we will add one soon.

However, since this `gpt2_bpe_tokenizer` is just a fall through to `torch.ops.torchtext`, so I think you can test it directly, without this API.

Reply:
Yes, I just added `tests/csrc` . We can use this directory to add any tests for csrc operators. I think to maintain a good standard of unit test coverage, we should add unit tests for most of the operators that we expose through `csrc`.